src/comments/comment.entity.ts

```diff
@@ -0,0 +1,74 @@
+import {
+  Column,
+  CreateDateColumn,
+  DeleteDateColumn,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+  UpdateDateColumn,
+} from 'typeorm';
+import { User } from '../users/users.legacy.entity';
+
+@Entity()
+export class Comment {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  content: string;
+
+  @Column()
+  commentableId: number;
+
+  @Column()
+  commentableType: 'answer' | 'comment' | 'question';
+
+  @Column()
+  agreeCount: number;
```

Comment:
You can derive this value through counting records meeting specific conditions in the following relation table. So you don't need to save in db.